.github/workflows/sycl-linux-precommit.yml

```
@@ -236,6 +237,57 @@ jobs:
       skip_run: ${{matrix.use_igc_dev && contains(github.event.pull_request.labels.*.name, 'ci-no-devigc') || matrix.skip_run || 'false'}}
       env: ${{ matrix.env || (contains(needs.detect_changes.outputs.filters, 'esimd') && '{}' || '{"LIT_FILTER_OUT":"ESIMD/"}') }}
 
+  E2E-with-new-offload-model:
+    needs: [build, detect_changes, compat_read_exclude]
+    if: |
+      !cancelled() &&
+      needs.build.outputs.build_conclusion == 'success' &&
+      contains(github.event.pull_request.labels.*.name, 'new-offload-model')
+    permissions:
+      contents: write
+      packages: read
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Intel / GEN 12 Integrated
+            runner: '["Linux", "gen12"]'
+            target_devices: opencl:cpu;level_zero:gpu
```

why not apply configuration that I suggested initially? Any objections? Again, I tried to cover as much variants as possible with the least number of configurations. In the current set, no combination is testing `opencl:gpu`, so if any test is filtered to only run on `opencl:gpu` we would miss it.
```suggestion
            target_devices: opencl:gpu;opencl:cpu
```